experiments/react-native-uifabric/package.json

```diff
@@ -7,7 +7,7 @@
   "typings": "lib/index.d.ts",
   "private": true,
   "scripts": {
-    "build": "just-scripts build --useMetro",
+    "build": "just-scripts build --useMetro --platform win32",
```

Comment:
Do we really want to build all the bundles during build?

The fluent repo split things out into:
build
lint
test
bundle
as top level commands.  That way you dont have to sit there waiting for everything to bundle whenever you do a build.

Reply:
OK, just pushed a new commit, which split the bundle task out of the build task.